content/docs/ref/check.md

```diff
@@ -4,21 +4,24 @@
 cml check create [options] <markdown report file>
 ```
 
-Similar to [`comment create`](/doc/ref/comment#create), but using GitHub's
-[checks interface](https://docs.github.com/en/rest/reference/checks).
+Posts a Markdown report using GitHub's
+[Checks API](https://docs.github.com/en/rest/reference/checks).
```

Comment:
Would https://docs.github.com/en/rest/guides/getting-started-with-the-checks-api be a better link?